test/sample_test_v2.yaml

```diff
@@ -372,6 +372,49 @@ spec:
             value: "{{inputs.parameters.namespace}}"
           - name: test-name
             value: "notebook-tfx"
+      - name: run-notebook-lightweight-tests
+        template: run-sample-tests
+        arguments:
+          parameters:
+          - name: test-results-gcs-dir
+            value: "{{inputs.parameters.test-results-gcs-dir}}"
+          - name: target-image-prefix
+            value: "{{inputs.parameters.target-image-prefix}}"
+          - name: dataflow-tft-image
```

Comment:
Do you think there is a way we can simplify image replacement? Maybe the `target-image-prefix` has enough information?

Reply:
By pushing the image URL to the scripts, we unnecessarily create a consistency between the sample test yamll and sample test script because they need to agree on the image name. Right?

Reply:
I was thinking about the consistency between image names used by component builders (`build_image.sh` files),  component definition (currently in sample python code), Cloud Build configs and test scripts/yamls. I think those image names must be consistent. The component builder and component definition should be the source of truth and the tests should not contain explicit image names/suffixes.

Reply:
>By pushing the image URL to the scripts
What exactly do you mean by "scripts"? The component definitions in the pipeline python files? Or do you mean the test scripts?

Reply:
The sample tests should care about the image tags because we current run the sample tests as submit. I can see if we move them to postsubmit, it should work fine by using the image tags that are consistent with others , such as cloud build.